### ISSUE TITLE
The style attribute of div elements that might surround images is removed.

### ISSUE THREAD

**Issue by gnotaras** [Automattic/amp-wp#358]
Hi all,

Sometimes plugins may add a `div` element, which has the `style` attribute set, around images _at runtime_. In the AMP version of the page, the `style` attribute of the surrounding `div` is stripped.

For testing purposes, manually add a `div` element around an image in the post editor. The image should be one that is attached to the post but without any caption set and without making use of the `caption` shortcode.

```
<div id="my-surrounding-div" style="width: 543px;">
    <img src="myimage.png" width="543" height="234" />
</div>
```

In the AMP version of the page the outer div becomes:

```
<div id="my-surrounding-div">
  ...
</div>
```

It would be nice if the style attribute was retained, at least its `width` property, which is important in cases in which, for example, a caption is added to the image on the fly (width is important for proper display of the caption).

George

**Comment by gnotaras** [comment]
Just noticed that this issue is related to #344

**Comment by westonruter** [comment]
This is fixed in 0.6.x in my testing. Inline styles get converted into style rules.
